.github/dependabot.yaml

```diff
@@ -0,0 +1,6 @@
+version: 2
```

Comment:
This is cool and it was proposed for other CSI repos too, I think this should come on its own PR

Reply:
Previously we didn't add similar stuff like this because of maintainability, it's easy to add things and make them work but if something fails e.g. if there's a dependency bump or a vuln notification then someone has to review that too and maintain this .github/<>.yaml files too. I think it was hard to find a volunteer to do this.

Reply:
I don't think `v1` is still supported. Also for a chart which doesn't exist yet I can't see a valid reason to use the old version?

I'm not quite sure of the context for the second question?

Reply:
is the last comment on the right thread? I think it should be in the Chart.yaml thread

Reply:
> Previously we didn't add similar stuff like this because of maintainability, it's easy to add things and make them work but if something fails e.g. if there's a dependency bump or a vuln notification then someone has to review that too and maintain this .github/<>.yaml files too. I think it was hard to find a volunteer to do this.

This bit?

Reply:
My bad, I maintain a few repos in kubernetes-csi but I forget that this repo is in kubernetes-sigs and doesn't have the same requirements as kubernetes-csi